assets/sass/local.sass

```diff
@@ -1,5 +1,12 @@
 @import "palette"
 
+div.govuk-width-container
```

Comment:
needs wider container width in Use of force incidents page
used 1170px instead of 1150px previously used to be same as new header width